packages/react-chat/src/components/Footer/index.tsx

```diff
@@ -44,8 +49,8 @@ const Footer: React.FC<FooterProps> = ({ withWatermark, hasEnded, onStart, onSen
   };
 
   return (
-    <Container withShadow={!!hasEnded} withWatermark={withWatermark}>
-      {hasEnded ? (
+    <Container withShadow={!!hasEnded || (!autostart && session.status === SessionStatus.IDLE)} withWatermark={withWatermark}>
+      {hasEnded || (!autostart && session.status === SessionStatus.IDLE) ? (
```

Comment:
I'd extract both of these boolean logic expressions into named variables for readability